src/ReduxToastr.js

```diff
@@ -33,11 +33,8 @@ class ReduxToastr extends Component {
     transitionOut: 'bounceOut',
     preventDuplicates: false,
     confirmOptions: {
-      transitionIn: 'bounceInDown',
-      transitionOut: 'bounceOutUp',
       okText: 'ok',
-      cancelText: 'cancel',
-      disableCancel: false
```

Comment:
the reason is simple - once a user's set his global config all default props are nullified(as default props are replaced - not merged). So the rest of the code better expect "undefined" state rather "false"